packages/xcm-util/README.md

```diff
@@ -9,8 +9,131 @@
 
 ## Installation
 
-TBD
+Before installing `xcm-util` package to your project, you need to setup Yarn 2, due to requirements of the `open-xcm-tools` packages.
+
+### Yarn 2 (Berry) installation
+
+First, set up a global Yarn, which will be used to create local instances:
+
+```bash
+npm install -g yarn
+```
+
+Then you need to locally change **Yarn** version for your project. In project folder:
+
+```bash
+yarn set version berry
+```
+
+After this actions you can use **Yarn** as usually (init project, add dependencies, etc.)
+
+### `xcm-util` installation
+
+```bash
+yarn add @open-xcm-tools/xcm-util
+```
 
 ## Usage
 
-TBD
+For showing all functionality let's take a look to the examples of `Location` and `Asset` entities:
```

Comment:
>For showing all functionality

"all functionality"? This package is rather diverse; I don't get what you mean here.